src/objects/core/zcl_abapgit_tadir.clas.abap

```diff
@@ -112,6 +116,31 @@ CLASS zcl_abapgit_tadir IMPLEMENTATION.
       ENDIF.
     ENDLOOP.
 
+    LOOP AT rt_tadir ASSIGNING <ls_tadir> WHERE obj_name(1) = '/'.
```

Comment:
suggest extracting logic to new private method

Reply:
I split `build` into smaller methods. the diff looks crazy but it's just refactoring.